src/common.ts

```diff
@@ -4,143 +4,105 @@ import { readFileSync, unlinkSync, writeFileSync } from 'fs';
 import { homedir } from 'os';
```

Comment:
Just to share, at some point (does not have to happen in this PR) I would extract the methods from this `common` class into purpose-specific modules. These `common` classes tend to be a "rubbish bin" to put all methods that a dev does not know where to put, and can be a nightmare of unrelated code in a future.

Reply:
Yes, I agree!